app/daos/UserDaoMongoDBImpl.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const Promise = require('bluebird');
+const User = require('../model/user.js');
+
+module.exports = function() {
+  const createUser = function(userData) {
+    const newUser = new User(userData);
+    return new Promise((resolve, reject) => {
+      newUser.save()
+        .then((user) => {
+          resolve(user.toObject());
+        })
+        .catch(reject);
+    });
+  };
+
+  return {
+    create: createUser
```

Comment:
Depends on the linter, but I think the latest JS allows trailing commas, which makes it easier to deal with diffs if new lines are added.